applications/visualizer/frontend/src/helpers/threeD/threeDHelpers.ts

```diff
@@ -0,0 +1,5 @@
+import { ViewerType, Visibility, type Workspace } from "../../models";
+
+export function getVisibleNeuronsInThreeD(workspace: Workspace): string[] {
```

Comment:
Could we have this method on `Workspace` instead? Like this, we can remove the `threeDHelper.ts` file